.circleci/config.yml

```diff
@@ -253,7 +253,7 @@ jobs:
         command: |
           ./bin/uat_deploy
 
-  auto_deploy_uat:
+  uat_master_deploy:
```

Comment:
Instead of renaming it, duplicate it.
```
auto_deploy_uat: 
  <<: *deploy_uat

uat_master_deploy:
  <<: *deploy_uat
```
This is because auto_deploy_uat doesn't always deploy to uat_master, it's dynamic and can deploy a pull request to uat e.g. ap-1xx-uat